ecosystem/sep-0024.md

```diff
@@ -170,26 +171,183 @@ There are several possible kinds of response, depending on whether the anchor ne
+
+If the anchor we're depositing to **doesn't support** claimable balances.
+
+1. Make a request to `/transactions/deposit/interactive` and provide the `claimable_balance_supported` request parameter.
+2. When the transaction status becomes `pending_user_transfer_start` send the required payment as described in the interactive webapp callback or the `/transaction` endpoint.
+3. The anchor will ignore the `claimable_balance_supported` field, discover the account does not have a trustline, and report transaction status as `pending_trust`.
+4. At this point both the wallet and anchor are waiting on the trustline to be established in order to continue the deposit payment. This is described here [steps 4-6](#stellar-account-doesnt-trust-asset)
+
+##### Wallet Feature: Claimable Balance Polling
+
+Polling allows the wallet to detect claimable balances and provide a UI to users that there is a balance available to claim. See the ["Claim Account Claimable Balance"](#wallet-feature-claim-account-claimable-balance) section to learn more about how to claim a balance.
+There are a few options as to how a wallet can poll and surface claimable balance information for their users.
+1. The wallet can poll the anchor's [transaction history endpoints](#anchor-feature-extend-transaction-history-endpoints-for-claimable-balances)
+
+2. The wallet can poll Horizon's [`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/list/) endpoint (or Horizon's [`/claimable_balances/`](https://developers.stellar.org/api/resources/claimablebalances/single/) single endpoint) to pull any available claimable balances for the claimant.
+
+   * The single endpoint needs the `claimable_balance_id` in order to retrieve the claimable balance info.
+     * This can be obtained by extracting the `claimable_balance_id` from `results_xdr` of the deposit transaction submitted if the anchor does not extend their [transaction history endpoints](#anchor-feature-extend-transaction-history-endpoints-for-claimable-balances)
+
+##### Wallet Feature: Claim Account Claimable Balance
+
+It's recommended that Wallets provides a UI to enable accounts to claim their balances [polled](#wallet-feature-claimable-balance-polling)
+
+[Example on how to claim a balance](https://developers.stellar.org/docs/glossary/claimable-balance/#claim-claimable-balance)
+Accounts will still need to establish trustline to the asset before claiming the outstanding balance.
+
+##### Anchor Claimable Balance Flow
+
+1. Wallets make a request to `/transactions/deposit/interactive` and provide the `claimable_balance_supported` request parameter.
+2. Anchors mark the transaction as [`claimable_balance_supported=true`](#deposit)
+3. Anchors detect that the user's Stellar account doesn't have a trustline.
+4. The anchor should then report transaction status as `pending_anchor` and submit a Stellar transaction containing a create claimable balance operation
```

Comment:
I think we can remove this line because:
1. The transaction status should already be in `pending_anchor` while the anchor checks for a trustline, and
2. The correct status once the anchor begins submitting the transaction is `pending_stellar`.